apps/metadata-editor/src/app/edit/edit-page.component.ts

```diff
@@ -151,6 +152,10 @@ export class EditPageComponent implements OnInit, OnDestroy {
       .subscribe((savedRecord) => {
         this.router.navigate(['edit', savedRecord.uniqueIdentifier])
       })
+
+    this.facade.record$.subscribe((record) => {
+      this.facade.hasRecordChangedSinceDraft(record)
+    })
```

Comment:
I think this could be wrapped into a `this.subscription.add()` to ensure unsubscribing.